docs/experimentConfigReadme.md

```diff
@@ -127,6 +128,7 @@ targets = [
         "visualSize" : [0.5, 0.5],              // 0.5m size
         "respawnCount" : 0,                     // Don't respawn
         "speed": [1.0, 3.0],                    // 1-3m/s speed
+        "symmetricEcc" : false;                 // Target will only spawn on one quadrant according to eccH/V signs
```

Comment:
As a convention we try to document the default values in these examples, so I would make this `True`